src/internet_identity/src/activity_stats.rs

```diff
@@ -103,5 +102,11 @@ pub fn update_active_anchors_stats(anchor: &Anchor, current_domain: &Option<IIDo
                 .get_or_insert_with(|| ActivityStats::new(time()));
             stats.update_counters(&context);
         }
+
+        // Active authn methods stats, II domains only
+        let stats = persistent_state
+            .active_authn_method_stats
+            .get_or_insert_with(|| ActivityStats::new(time()));
+        stats.update_counters(&current_device);
```

Comment:
I see that in `AuthnMethodCounter::update_counters()` below we do an internal check whether on II domain (so the code here is correct), but the difference with the impl. of `DomainActiveAnchorCounter::update_counters()` (where we have a comment    `/// Only called if `current_domain` corresponds to an II domain.`) is confusing, especially that both cases are commented with "II domains only", but for the active anchor stats above we have the `if II domain..` and here not.  Maybe change `AuthnMethodCounter::update_counters()` so that `if ..` is not needed?

Reply:
Ah, yes. Good point, changing that now. 👍